### ISSUE TITLE
"node.exe exited with code 1" in VS2017 after updating from TS 3.1.1 to TS 3.3.1

### ISSUE THREAD

**Issue by KarloX2** [Microsoft/TypeScript#30580]
**TypeScript Version:**  3.3.1

Hi!
My project won't compile any more in VS2017 after updating from TS 3.1.1 to 3.3.1

The first error that appears in the error window is this:

`"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\VisualStudio\NodeJs\node.exe" exited with code 1.`

Then there come a lot more errors, most of which are like `Cannot find name xxx`. I guess these errors appear as a consequence of the error above(?)

In the output window I also find these lines:

```
2>Task "VsTsc"
2>  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\VisualStudio\NodeJs\node.exe "C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js"  --project "C:\Projects\MyProject\tsconfig.json" --listEmittedFiles --locale en-US --listFiles --noEmit
2>  Unknown output:                 throw e;
2>  Unknown output:                 ^
2>  Unknown output:     at checkNestedBlockScopedBinding (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:46274:48)
2>  Unknown output:     at checkIdentifier (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:46180:13)
2>  Unknown output:     at checkExpressionWorker (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:52182:28)
2>  Unknown output:     at checkExpression (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:52154:42)
2>  Unknown output:     at checkNonNullExpression (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:48529:37)
2>  Unknown output:     at checkPropertyAccessExpressionOrQualifiedName (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:48562:28)
2>  Unknown output:     at checkPropertyAccessExpression (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:48555:20)
2>  Unknown output:     at checkExpressionWorker (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:52211:28)
2>  Unknown output:     at checkExpression (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:52154:42)
2>  Unknown output:     at checkNonNullExpression (C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.3\tsc.js:48529:37)
2>Done executing task "VsTsc".
```

By changing the project properties to use 3.1 instead of the latest TS the error doesn't appear any more.

What can I do?

Thanks!

**Comment by RyanCavanaugh** [comment]
Looks like a duplicate of #29524

**Comment by KarloX2** [comment]
Thanks, @RyanCavanaugh!

Is there something like a nightly build (installer for VS2017 or a way to exchange binaries) that I can use to verify that this error will be resolved by the fix already filed with the other ticket?

**Comment by KarloX2** [comment]
The "node.exe exited with code 1" problem is resolved after installing TS 3.4 RC. I still have a number of other errors messages when I compile my code that used to work fine with TS 3.1, but that seems to be a different issue. So I close this ticket.
Thanks!!

**Comment by meisamdev** [comment]
this can help you

https://stackoverflow.com/questions/57034700/msb6006-node-exe-exited-with-code-1-in-vs2019
